build.sh

```diff
@@ -103,6 +103,22 @@ for i in "${ARGS[@]}"; do
 	esac;
 done
 
+if ! command -v /bin/bear &> /dev/null; then
```

Comment:
not sure if this (re-exec) shouldn't be a first thing in the file. If  `build.project` would contain eg. `CFLAGS="$CFLAGS -O2"` we will append `-O2` twice (source + export + re-exec + second source)

Reply:
also: don't assume `bear` is installed in `bin`